src/core/etl/src/Flow/ETL/Row/Factory/NativeEntryFactory.php

```diff
@@ -330,6 +334,10 @@ private function isUuid(string $string) : bool
 
     private function isXML(string $string) : bool
     {
+        if ('<' !== $string[0]) {
```

Comment:
I think we might want trim string just in case of some empty spaces at the beginning.

Reply:
But do you mean in all of that check or globally? Like `$value = \trim($value)` before passing it to any of the checks:
```php
        if (\is_string($value)) {
            $value = \trim($value);

            if ('' !== $value) {
                if ($this->isJson($value)) {
                // ...

Reply:
`$this->isXML(trim($string))` - we don't want to trim values that should not be trimmed as spaces at the beginning might be expected in some scenarios